src/Http/Resources/QuestionAnswerResource.php

```diff
@@ -18,6 +18,7 @@ public function toArray($request)
             'id' => $this->id,
             'user_id' => $this->user_id,
             'question_id' => $this->question_id,
+            'questionnaire_model_id' => $this->questionnaire_model_id,
```

Comment:
Questionnaire Answer has `questionnaire_model_id` field? Why?
Also, this resource is probably not used anywhere (yet)

Reply:
this is use in QuestionnaireAnswerService
answer is related to questionnaire model because user can get the same question for more then 1 course